src/pages/VoterDetail/Holding/index.tsx

```diff
@@ -0,0 +1,88 @@
+/** @jsxImportSource @emotion/react */
+import React, { useMemo } from 'react';
+import BigNumber from 'bignumber.js';
+import { Paper, Typography } from '@mui/material';
+import { Delimiter, LabeledProgressBar, Icon } from 'components';
+import PLACEHOLDER_KEY from 'constants/placeholderKey';
+import { useTranslation } from 'translation';
+import { convertWeiToTokens } from 'utilities';
+import { useStyles } from './styles';
+
+interface IHoldingProps {
+  className?: string;
+  balanceWei: BigNumber | undefined;
+  delegateCount: number | undefined;
+  votesWei: BigNumber | undefined;
+  delegating: boolean;
+}
+
+export const Holding: React.FC<IHoldingProps> = ({
+  className,
+  balanceWei,
+  delegateCount,
+  votesWei,
+  delegating,
+}) => {
+  const styles = useStyles();
+  const { t } = useTranslation();
+
+  const readableVenusBalance = useMemo(() => {
+    if (!balanceWei) {
+      return PLACEHOLDER_KEY;
+    }
+    return convertWeiToTokens({
+      valueWei: balanceWei,
+      tokenId: 'xvs',
+      addSymbol: false,
+      returnInReadableFormat: true,
+    });
+  }, [balanceWei]);
+
+  const readableVotes = useMemo(() => {
+    if (!votesWei) {
+      return PLACEHOLDER_KEY;
+    }
+    return convertWeiToTokens({
+      valueWei: votesWei,
+      tokenId: 'xvs',
+      addSymbol: false,
+      returnInReadableFormat: true,
+    });
+  }, [votesWei]);
```

Comment:
You can use `useConvertWeiToReadableTokenString` for both of these. I'd also suggest grabbing the id of XVS from the `TOKENS` constant (`import { TOKENS } from 'constants/tokens';`)